apps/src/netsim/dataConverters.js

```diff
@@ -104,12 +110,11 @@ exports.alignDecimal = function (decimalString) {
     return prev;
   }, 0);
 
-  var nbspChar = "\xA0";
-  var nbspPadding = new Array(mostDigits + 1).join(nbspChar);
+  var zeroPadding = new Array(mostDigits + 1).join('0');
 
   return numbers.map(function (numString) {
     // Left-pad each number with non-breaking spaces up to max width.
```

Comment:
This looks like a good candidate to extract into a util function if it isn't already one.